packs/feats/you-dont-smell-right.json

```diff
@@ -28,7 +28,33 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Smell disguise",
```

Comment:
Don't forget to localise this. I'd maybe go with something more descriptive like "Seek to determine if a creature is disguised" or something along those lines

Reply:
Added localization and expanded on the description.